partner_firstname/models.py

```diff
@@ -38,12 +38,16 @@ class ResPartner(models.Model):
         required=False,
         store=True)
 
+    @api.model
+    def _get_computed_name(self, lastname, firstname):
+        """Write the 'name' field according to splitted data."""
+        return u" ".join((p for p in (lastname, firstname) if p))
```

Comment:
Didn't we talk about putting a comma in the middle to correctly write the form "Lastname, Firstname"?